src/pages/api/profile/user-profile/index.ts

```diff
@@ -113,7 +113,7 @@ const patchHandler: Handler = async (request, response) => {
       );
 
       response.statusCode = HttpStatus.OK;
-      response.json({ updatedUserProfile });
+      response.json(updatedUserProfile);
```

Comment:
And to match the REST convention, the endpoint now returns the profile as the data, and not an object with a `userProfilePayload` anymore.